lib/orchestrator/core/mixin.rb

```diff
@@ -3,6 +3,12 @@
 module Orchestrator
     module Core
         module Mixin
+            # Returns the module id as defined in the database.
+            #
+            # @return [Symbol] the id of the module instance
+            def id
+                @__config__.settings.id.to_sym
```

Comment:
I would actually return this as a string

Reply:
It's being sym'ed for consistency with [`system.id`](https://github.com/acaprojects/ruby-engine/blob/master/lib/orchestrator/core/system_proxy.rb#L136-L141).

Can definitely go with a string here, but then that raises the question: should the system id accessor switch to a string too?

Reply:
You are right and it already exists as a symbol so let's do this :)